node-operators/networks/collators/activities.md

```diff
@@ -25,24 +25,27 @@ description: 关于深入了解并学习成为Moonbeam网络中收集人相关
-    |        减少自身委托量         |       {{ networks.moonriver.collator_timings.can_bond_less.rounds }}轮次（{{ networks.moonriver.collator_timings.can_bond_less.hours }}小时）       |
-    | 奖励发放（在本轮结束后） |    {{ networks.moonriver.delegator_timings.rewards_payouts.rounds }}轮次（{{ networks.moonriver.delegator_timings.rewards_payouts.hours }}小时）    |
+    |            撤销委托             | {{ networks.moonriver.delegator_timings.revoke_delegations.rounds }}轮次（{{ networks.moonriver.delegator_timings.revoke_delegations.hours }}小时） |
+    |          减少自身委托量          |       {{ networks.moonriver.collator_timings.can_bond_less.rounds }}轮次（{{ networks.moonriver.collator_timings.can_bond_less.hours }}小时）       |
+    |      奖励发放（在本轮结束后）      |    {{ networks.moonriver.delegator_timings.rewards_payouts.rounds }}轮次（{{ networks.moonriver.delegator_timings.rewards_payouts.hours }}小时）    |
+    |           最大离线轮次           |         {{ networks.moonbeam.collator_timings.max_offline.rounds }} rounds ({{ networks.moonbeam.collator_timings.max_offline.hours }} hours)         |        
 
 === "Moonbase Alpha"
     |               变量                |                                                                         值                                                                         |
     |:-------------------------------------:|:-----------------------------------------------------------------------------------------------------------------------------------------------------:|
-    |            轮次时长             |                        {{ networks.moonbase.staking.round_blocks }}区块（{{ networks.moonbase.staking.round_hours }}小时）                        |
-    |           离开候选人            |    {{ networks.moonbase.collator_timings.leave_candidates.rounds }}轮次（{{ networks.moonbase.collator_timings.leave_candidates.hours }}小时）    |
+    |            轮次时长          |                        {{ networks.moonbase.staking.round_blocks }}区块（{{ networks.moonbase.staking.round_hours }}小时）                        |
+    |          离开候选人          |    {{ networks.moonbase.collator_timings.leave_candidates.rounds }}轮次（{{ networks.moonbase.collator_timings.leave_candidates.hours }}小时）    |
     |           撤销委托           | {{ networks.moonbase.delegator_timings.revoke_delegations.rounds }}轮次（{{ networks.moonbase.delegator_timings.revoke_delegations.hours }}小时） |
-    |      减少自身委托量      |      {{ networks.moonbase.delegator_timings.del_bond_less.rounds }}轮次（{{ networks.moonbase.delegator_timings.del_bond_less.hours }}小时）      |
-    | 奖励发放（在本轮结束后） |    {{ networks.moonbase.delegator_timings.rewards_payouts.rounds }}轮次（{{ networks.moonbase.delegator_timings.rewards_payouts.hours }}小时）    |
+    |        减少自身委托量         |      {{ networks.moonbase.delegator_timings.del_bond_less.rounds }}轮次（{{ networks.moonbase.delegator_timings.del_bond_less.hours }}小时）      |
```

Comment:
```suggestion
    |        减少自身委托量         |      {{ networks.moonbase.delegator_timings.can_bond_less.rounds }}轮次（{{ networks.moonbase.delegator_timings.can_bond_less.hours }}小时）      |
```

great catches! just noticed this should be can_bond_less though, i'll need to fix that on the english site!